streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -1740,13 +1740,14 @@ public void shouldComputeOffsetSumFromCheckpointFileForUninitializedTask() throw
         taskManager.handleRebalanceStart(singleton("topic"));
         final StateMachineTask uninitializedTask = new StateMachineTask(taskId00, taskId00Partitions, true);
         expect(activeTaskCreator.createTasks(anyObject(), eq(taskId00Assignment))).andStubReturn(singleton(uninitializedTask));
-        expect(standbyTaskCreator.createTasks(anyObject())).andStubReturn(Collections.emptySet());
-        replay(activeTaskCreator, standbyTaskCreator);
+        replay(activeTaskCreator);
         taskManager.handleAssignment(taskId00Assignment, emptyMap());
 
         assertThat(uninitializedTask.state(), is(State.CREATED));
 
         assertThat(taskManager.getTaskOffsetSums(), is(expectedOffsetSums));
+
+        Mockito.verify(standbyTaskCreator).createTasks(Collections.emptyMap());
```

Comment:
I do not think you need this verification since also with EasyMock the call is not verified (i.e., `verify()` on the mock is never called).

Reply:
To be honest, here I followed what was done in PR https://github.com/apache/kafka/pull/13681
For example, the original in the mentioned PR was.
```
...
expect(activeTaskCreator.createTasks(consumer, Collections.emptyMap())).andReturn(emptySet());
...
```
(https://github.com/apache/kafka/pull/13681/files#diff-48bc1476f0437fd711093c7c80ce73eda10be0511705799b4248545c203d521dL294)


and with Mockito we changed it to
```
...
Mockito.verify(activeTaskCreator).createTasks(consumer, Collections.emptyMap());
...
```
(https://github.com/apache/kafka/pull/13681/files#diff-48bc1476f0437fd711093c7c80ce73eda10be0511705799b4248545c203d521dR306)

I followed the same logic for what I thought was a stronger verification, but I am happy to remove it if you think it is unnecessary. I realise the reasoning for doing the above in that PR might have made more sense because there was more than one argument.

Let me know!

Reply:
Oh, I see what you mean. You mean that in the EasyMock version there are no calls to
```
verify(standbyTaskCreator)
```
in the these specific tests.
Yeah, sure, I will change it!